rosidl_generator_tests/CMakeLists.txt

```diff
@@ -34,6 +34,7 @@ if(BUILD_TESTING)
     ${test_interface_files_ACTION_FILES}
     ${test_interface_files_MSG_FILES}
     ${test_interface_files_SRV_FILES}
+    msg/BasicIdl.idl
```

Comment:
Just to be totally clear, this is an unrelated test, correct?  I'm actually fine with adding it in here for testing purposes, but I just want to make sure I'm not missing something.

Reply:
I just wanted to make extra sure, here in `rosidl` repo, that the new changes didn't break anything for `.idl` files, which are handled slightly differently as they don't go through `rosidl_adapter`. The intention for this extra file is to serve that purpose, since there were not yet any `.idl` files being tested by this package.